src/schemathesis/cli/handlers.py

```diff
@@ -9,3 +13,15 @@ def handle_event(self, context: ExecutionContext, event: events.ExecutionEvent)
     def shutdown(self) -> None:
         # Do nothing by default
         pass
+
+
+def _get_unique_failures(checks: List[SerializedCheck]) -> List[SerializedCheck]:
```

Comment:
Since it is used in other modules, I think the leading underscore in the function name is not needed